packages/apps-config/src/api/spec/index.ts

```diff
@@ -54,6 +54,8 @@ import trustbase from './trustbase';
 import uniarts from './uniarts';
 import zenlink from './zenlink';
 import zero from './zero';
+import sgc from './sgc';
```

Comment:
Please add this alphabetical, as well as below where it is used. (It really is simpler to maintain, there are a lot)